ttnn/cpp/ttnn/operations/eltwise/binary/binary_pybind.hpp

```diff
@@ -253,6 +253,114 @@ void bind_binary_composite_with_rtol_atol(py::module& module, const binary_opera
             py::arg("memory_config") = std::nullopt});
 }
 
+template <typename binary_operation_t>
+void bind_binary_composite_with_overload(py::module& module, const binary_operation_t& operation, const std::string& description) {
```

Comment:
can we make this bind_binary_composite_with_overload a op specific name if it used by a common set of ops? bind_div_ops ?